File: ChatSockets/persistence/MessageRepositoryMock.cs
using System.Collections.Generic;
using chat.model;

namespace chat.persistence
{
    public class MessageRepositoryMock:IMessageRepository
    {
        public void save(Message entity)
        {
        }

        public void delete(int id)
        {
        }

        public Message findOne(int id)
        {
            return null;
        }

        public void update(int id, Message e)
        {

        }

        public IEnumerable<Message> getAll()
        {
            return null;
        }
    }
}

File: ChatProtobuff_v3/ChatPersistence/src/main/java/chat/persistence/repository/mock/MessageRepositoryMock.java
package chat.persistence.repository.mock;

import chat.model.Message;
import chat.persistence.MessageRepository;

public class MessageRepositoryMock implements MessageRepository {
    @Override
    public void save(Message message) {

    }

    @Override
    public void delete(Integer integer) {

    }

    @Override
    public Message findOne(Integer integer) {
        return null;
    }

    @Override
    public void update(Integer integer, Message message) {

    }

    @Override
    public Iterable<Message> getAll() {
        return null;
    }
}
